src/main/resources/api.yaml

```diff
@@ -0,0 +1,174 @@
+swagger: '2.0'
+info:
+  description: >-
+    This documents the OTP Middleware endpoints. For this sample, you can use
+    the api key `special-key` to test the authorization filters.
+  version: 0.0.0
```

Comment:
Is this file being generated manually? Either way, it would be good to have a comment at the top of this file describing its purpose (e.g., how it should be maintained, where it is used, etc.). Also, I worry about the version becoming out of sync with what semantic-release labels things. Although, perhaps the API version and the software version don't necessarily match up?

Reply:
This file is generated manually for now...